Bottom: 86c88656023e5c0ef90c54716400c2b1d571aa24
Top:    3479c7c6032e318fc07b9e92d516a537808c77d8
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-08 20:32:28 +0000

Refresh of more-work-on-trying-to-speed-u

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 9f4b142..7fe8c4a 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1330,7 +1330,7 @@ PutContour (jmp_buf * e, PLINE * cntr, POLYAREA ** contours, PLINE ** holes,
         printf ("PATH 1\n");
         r_delete_entry (owner->contour_tree, (BoxType *)cntr);
       }
-      printf ("Put contour adding a brand new contour\n");
+      printf ("Put contour adding a new contour\n");
       InsCntr (e, cntr, contours);
     }
   /* put hole into temporary list */
@@ -1341,6 +1341,7 @@ PutContour (jmp_buf * e, PLINE * cntr, POLYAREA ** contours, PLINE ** holes,
 	{
 	  cntr->next = parent_contour->next;
 	  parent_contour->next = cntr;
+          printf ("Put Contour adding a hole directly to its parent\n");
           if (owner != parent)
             {
               if (owner != NULL) {
@@ -1352,9 +1353,9 @@ PutContour (jmp_buf * e, PLINE * cntr, POLYAREA ** contours, PLINE ** holes,
 	}
       else
 	{
-          printf ("Put contour adding a hole\n");
 	  cntr->next = *holes;
 	  *holes = cntr;	/* let cntr be 1st hole in list */
+          printf ("Put Contour adding a hole\n");
           /* We don't insert the holes into an r-tree,
            * they just form a linked list */
           if (owner != NULL) {
